jstests/sharding/query/lookup/lookup.js

```diff
@@ -25,7 +32,8 @@ function compareId(a, b) {
 
 // Helper for testing that pipeline returns correct set of results.
 function testPipeline(pipeline, expectedResult, collection) {
```

Comment:
This was silently never actually checking the results in a meaningful way because arrayEq returns true/false